examples/fair_prep_results.py

```diff
@@ -0,0 +1,141 @@
+from fp.post_processors import NoPostProcessing,RejectOptionPostProcessing, EqualOddsPostProcessing, CalibratedEqualOddsPostProcessing
+from fp.pre_processors import NoPreProcessing, DIRemover, Reweighing
+
+import numpy as np
+get_ipython().run_line_magic('matplotlib', 'inline')
+import matplotlib.pyplot as plt
+
+seeds = [0xbeef, 0xcafe, 0xdead, 0xdeadcafe, 0xdeadbeef, 0xbeefcafe, 0xcafebeef, 50, 583, 5278, 100000, 0xefac,0xfeeb, 0xdaed, 0xefacdaed, 0xfeebdead]
+samplers = [CompleteData()]
+missings = [CompleteCaseAnalysis()]
+scalers = [NamedMinMaxScaler(), NamedStandardScaler()]
+learners = [NonTunedLogisticRegression(), LogisticRegression(), NonTunedDecisionTree(), DecisionTree()]
+post_processors = [NoPostProcessing(),RejectOptionPostProcessing(), CalibratedEqualOddsPostProcessing()]
+pre_processors = [NoPreProcessing(), DIRemover(1.0), DIRemover(0.5), Reweighing()]
+
+path = os.path.join(os.getcwd(), "logs")
+
+def extract_info(present_acc,present_di,present_fnr,present_fpr):
+    '''
+    Function to extract the metrics (accuracy, DI, false_positive_rate, false_negative_rate)value from the log files 
+    '''
+    try:
+        for folderName, subfolders, filenames in os.walk(path):      
+            for subfolder in subfolders:
+                filelist = os.listdir(path+'/'+subfolder)
+                for f in filelist:
+                    file_path = path+'/'+subfolder+'/'+f
+                    with open(file_path,'r+') as f:
+                        for line in f:
+                            
+                            if line.startswith("test,,disparate_impact"):
+                                di = line.split(',')
+                                di_value = di[-1].rstrip("\n")
+                                present_di.append(float(di_value))
+                            
+                            elif line.startswith("test") and 'accuracy' in line and 'None' in line:
+                                acc = line.split(',')
+                                acc_val = acc[-1].rstrip("\n")
+                                present_acc.append(float(acc_val))
+                            
+                            elif line.startswith("test") and 'generalized_false_positive_rate' in line and 'None' in line:
+                                fnr = line.split(',')
+                                fnr_val = fnr[-1].rstrip("\n")
+                                present_fnr.append(float(fnr_val))
+                            
+                            elif line.startswith("test") and 'generalized_true_positive_rate' in line and 'None' in line:
+                                fpr = line.split(',')
+                                fpr_val = fpr[-1].rstrip("\n")
+                                present_fpr.append(float(fpr_val))
+                shutil.rmtree(path+'/'+subfolder)            
+    except:
+        pass
+    
+    return present_acc,present_di,present_fnr,present_fpr
+
+def exp_run_pre_pro_param(seed,learn,pre,post):
```

Comment:
give understandable names to functions please

Reply:
changed.